oss_src/unity/python/sframe/cython/cy_sarray_builder.pxd

```diff
@@ -0,0 +1,43 @@
+'''
```

Comment:
I actually started in the SDK, but after talking with Hoyt about taking a list of types as a parameter to several functions, it seemed like cython was the best route.